### ISSUE TITLE
Nothing displayed

### ISSUE THREAD

**Issue by carrbrpoa** [David-Desmaisons/Vue.D3.tree#16]
Hello,

Tried to display a tree with sample data. I see the `svg` element is sized, but nothing is displayed. Code:

```
<template>
    <div>
        <tree class="tree" :identifier="getId" :node-text="text" :data="treeData" :type="'tree'" :duration="5" :margin-y="150"> </tree>
    </div>
</template>
<script>
    import { tree } from 'vued3tree';

    export default {
        components: {
            tree
        },
        data() {
            return {
                text: 'text',
                treeData: {
                    id: 1,
                    text: 'father',
                    children: [
                        {
                            id: 2,
                            text: 'son1',
                            children: [
                                { id: 4, text: 'grandson' },
                                { id: 5, text: 'grandson2' }
                            ]
                        },
                        {
                            id: 3,
                            text: 'son2',
                            children: [
                                { id: 6, name: 'grandson3' },
                                { id: 7, text: 'grandson4' }
                            ]
                        }
                    ]
                }
            };
        },
        methods: {
            getId(node) {
                return node.id;
            }
        }
    };
</script>

<style scoped>
    .tree {
        max-height: 600px;
        width: 100%;
    }
</style>
```

Any ideas? Thanks!

**Comment by David-Desmaisons** [comment]
Try to set a hight:

.tree {
        height: 600px;
        width: 100%;
    }

**Comment by qianbi** [comment]
I have the same problem;
Through debug, I found that without setting Marginx, the transform attribute of svg>g was "translate(06,20)". When Marginx:20px is set, the transform attribute of svg>g is "translate(206,20)".

**Comment by curtisdelicata** [comment]
All I see in Vue console is emitted events with the payload. Nothing is rendered.

**Comment by curtisdelicata** [comment]
I tried adding margin-x and margin-y but didn't help?

**Comment by curtisdelicata** [comment]
With some help, I got this working. The tree component doesn't show the text for one out of four labels but it renders fine. 

https://github.com/modularsoftware/genealogy/blob/master/resources/assets/js/pages/trees/Show.vue
https://github.com/modularsoftware/genealogy/blob/master/resources/assets/js/pages/trees/Edge.vue

**Comment by snoop244** [comment]
I'm not clear on how to get the op's code to work.  I'm having the same problem: <D3Tree> is not visible on the page, though it shows in Vue DevTools Components.

**Comment by David-Desmaisons** [comment]
@qianbi and @curtisdelicata I just released a 3.5.1 version where I fix a bug when zoomable was false that may be the same as the one you describe.
Please let me know if it solve the problem if not cfreate a webpackbin so I can investigate.
